Meadowlark-js/packages/meadowlark-core/src/metaed/MetaEdValidation.ts

```diff
@@ -66,17 +67,20 @@ export function validateEntityBodyAgainstSchema(metaEdModel: TopLevelEntity, bod
 }
 
 /**
- * Validates that those properties which are present actually belong in the body.
+ * Validates that those queryParameters which are present actually belong in the MetaEd entity.
  */
-export function validatePartialEntityBodyAgainstSchema(metaEdModel: TopLevelEntity, body: object): string[] {
-  const validationResult: Joi.ValidationResult = metaEdModel.data.meadowlark.joiSchema.validate(body, {
+export function validateQueryParametersAgainstSchema(
```

Comment:
As part of the splitting, I split out query parameter validation into its own middleware. Renamed things down the line to be consistent.